src/screens/Assets/Assets.js

```diff
@@ -189,20 +190,22 @@ class AssetsScreen extends React.Component<Props, State> {
     const {
       fetchInitialAssets,
       assets,
+      navigation,
+      logScreenView,
     } = this.props;
 
-    Answers.logContentView('Assets screen');
+    logScreenView('assets', 'Assets');
```

Comment:
On Contact screen you are passing activeTab as a first parameter (view). This screen also has tabs (assets and collectibles), so I'm wondering maybe this data should be logged for this screen as well.